scss/chart.scss

```diff
@@ -961,6 +961,14 @@ body {
             line-height: 1.2;
 
             margin-bottom: 4px;
+
+          .popup-item-val a {
+            text-decoration: none;
+          }
+          .popup-item-val a:hover {
+            border-bottom: 1px solid;
+            color: #0000EE;
```

Comment:
The typical blue link color is too dark for dark mode.
![Capture d’écran, le 2021-04-09 à 19 45 55](https://user-images.githubusercontent.com/824534/114251150-fec8f580-996d-11eb-9c98-fea590703bbc.jpg)

I suggest changing it so it stays the same color and just adds an underline:
```
border-bottom: 1px solid;
color: #0000EE;
```
to
```
text-decoration: underline;
color: inherit;
```